test/ProgramScopeConstants/in_storage_buffer_descriptor_map.cl

```diff
@@ -1,4 +1,4 @@
-// RUN: clspv %target %s -o %t.spv -module-constants-in-storage-buffer -int8=0
```

Comment:
@rjodinchr observed that Clang is inserting explicit padding members.

In this case we now have  an i8 first member, a [3 x i8] inserted second member.  Everything breaks when artificially trying to remap i8 to u32.  Use the default, which is actually support i8.